apps/frontend/src/libs/components/header/libs/components/user-popover/user-popover.tsx

```diff
@@ -0,0 +1,42 @@
+import { Popover } from "~/libs/components/components.js";
+import { useAppDispatch, useCallback } from "~/libs/hooks/hooks.js";
+import { actions as authActions } from "~/modules/auth/auth.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	children: React.ReactNode;
+	email: string;
+	name: string;
+};
+
+const UserPopover = ({ children, email, name }: Properties): JSX.Element => {
+	const dispatch = useAppDispatch();
+
+	const handleLogout = useCallback((): void => {
+		void dispatch(authActions.logout());
+	}, [dispatch]);
+
+	return (
+		<Popover
+			content={
+				<div className={styles["user-popup"]}>
```

Comment:
It's not a big deal, but
```suggestion
				<div className={styles["user-popover"]}>
```

Reply:
Oh, yeah, thanks, forgot about it)